railties/lib/rails/generators/rails/plugin/templates/test/test_helper.rb.tt

```diff
@@ -10,11 +10,6 @@ ActiveRecord::Migrator.migrations_paths << File.expand_path("../db/migrate", __d
 <% end -%>
 require "rails/test_help"
 
-<% unless engine? -%>
-require "rails/test_unit/reporter"
-Rails::TestUnitReporter.executable = "bin/test"
```

Comment:
These lines are already included in `rails/plugin/test`:

https://github.com/rails/rails/blob/56d8ff83726bb6c0b1f570deb057835f90a75e30/railties/lib/rails/plugin/test.rb#L4-L6

Which is required by `bin/test`:

https://github.com/rails/rails/blob/56d8ff83726bb6c0b1f570deb057835f90a75e30/railties/lib/rails/generators/rails/plugin/templates/bin/test.tt#L4